tcms/xmlrpc/api/testcase.py

```diff
@@ -286,18 +286,19 @@ def create(values, **kwargs):
 
         # Add tag to the case
         for tag in string_to_list(values.get('tag', [])):
-            t, c = Tag.objects.get_or_create(name=tag)
-            tc.add_tag(tag=t)
+            tag, _ = Tag.objects.get_or_create(name=tag)
```

Comment:
again, doesn't pylint complain that you are overriding/shadowing the `tag` variable from the loop condition ?

Reply:
Same as above